ldap/servers/plugins/sync/sync_persist.c

```diff
@@ -1084,7 +1084,7 @@ sync_send_results(void *arg)
 
     slapi_ch_free((void **)&req->req_orig_base);
     slapi_filter_free(req->req_filter, 1);
-    sync_cookie_free(&req->req_cookie);
```

Comment:
I'm not very familiar with this area of code... Could you please explain why we don't need to `sync_cookie_free` here?

Reply:
As I understand it is because its lead to a double free as the cookie is created and freed by the caller function.